flair/data.py

```diff
@@ -237,7 +237,7 @@ def set_embedding(self, name: str, vector: torch.tensor):
         device = flair.device
         if len(self._embeddings.keys()) > 0:
             device = next(iter(self._embeddings.values())).device
-        self._embeddings[name] = vector.to(device, non_blocking=True)
+        self._embeddings[name] = vector.to(device)
```

Comment:
I'm generally not yet sure if sending embeddings to devices with `non_blocking=True` makes sense or not, but the current version does set `non_blocking`, which would be removed with this PR. Do you think it makes no sense here?

Reply:
In my measures, without this LRU cache, non_blocking set to True was slightly slower (on inference) than when it was set to True. I was wondering why it was set to True plus why it is slower because https://discuss.pytorch.org/t/should-we-set-non-blocking-to-true/38234 says that it should be approximately the same when not required. Therefore I thought, it s not the default (kind of recommended) option, it s slightly slower, I remove. However, I may have missed something. 
To finish, my understanding of the description of the option and from when this line is called, I would have bet for a slight performance improvement when the option is set to True as the data are not used just after, so Pytorch can be lazy here. Anyway, with this cache, loading time are reduced a lot so not sure if it changes anything.

Reply:
ok great - I haven't measured the difference so if its faster without `non_blocking` then lets remove it :)